test/core/referencesImport.js

```diff
@@ -0,0 +1,57 @@
+var Transformer = require("../../lib/babel/transformation/transformer");
+var transform   = require("../../lib/babel/transformation");
+var babel       = require("../../lib/babel/api/node");
+var chai        = require("chai");
+
+suite("traversal path", function () {
+  test("referencesImport: default import", function () {
+    var expectCode = "import foo from 'bar'; foo();";
+
+    var actualCode = transform(expectCode, {
+      blacklist: "strict",
+      plugins: [new Transformer("foobar", {
+        CallExpression: function () {
+          if (this.get('callee').referencesImport('bar', 'default')) {
+            return "worked()";
+          }
+        }
+      })]
+    }).code;
+
+    chai.expect(actualCode).to.contain("worked();");
+  });
+
+  test("referencesImport: named import", function () {
+    var expectCode = "import { foo as woo } from 'bar'; woo();";
+
+    var actualCode = transform(expectCode, {
+      blacklist: "strict",
+      plugins: [new Transformer("foobar", {
+        CallExpression: function () {
+          if (this.get('callee').referencesImport('bar', 'foo')) {
+            return "worked()";
+          }
+        }
+      })]
+    }).code;
+
+    chai.expect(actualCode).to.contain("worked();");
+  });
+
+  test("referencesImport: namespace", function () {
+    var expectCode = "import * as foo from 'bar'; foo.wow();";
+
+    var actualCode = transform(expectCode, {
+      blacklist: "strict",
+      plugins: [new Transformer("foobar", {
+        CallExpression: function () {
+          if (this.get('callee').referencesImport('bar', '*')) {
```

Comment:
This test is currently failing; what is the correct way to check the import here?